wyvern/experimentation/providers/eppo_provider.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+from typing import List
+
+import eppo_client  # type: ignore
+from eppo_client.assignment_logger import AssignmentLogger  # type: ignore
+from eppo_client.config import Config  # type: ignore
+
+from wyvern import request_context
+from wyvern.components.events.events import LoggedEvent
+from wyvern.config import settings
+from wyvern.event_logging import event_logger
+from wyvern.experimentation.experimentation_logging import (
+    ExperimentationEvent,
+    ExperimentationEventData,
+)
+from wyvern.experimentation.providers.base import BaseExperimentationProvider
+
+
+class EppoExperimentationClient(BaseExperimentationProvider):
+    """
+    An experimentation client specifically for the Eppo platform.
+
+    Extends the BaseExperimentationProvider to provide functionality using the Eppo client.
+
+    Methods:
+    - __init__() -> None
+    - get_result(experiment_id: str, entity_id: str, **kwargs) -> str
+    - log_result(experiment_id: str, entity_id: str, variant: str) -> None
+    """
+
+    def __init__(self):
+        # AssignmentLogger is a dummy logger that does not log anything.
+        # We handle logging ourselves in the log_result method.
+        client_config = Config(
+            api_key=settings.EPPO_API_KEY,
+            assignment_logger=AssignmentLogger(),
```

Comment:
do we have to pass assignment_logger=AssignmentLogger() even if we don't use it?

Reply:
oh hey check this out: https://docs.geteppo.com/feature-flags/sdks/python#define-an-assignment-logger-experiment-assignment-only

shall we write a `class WyvernAssignmentLogger(AssignmentLogger)` where the log_result actually happens. As a result, we don't really need to define a `def log_result` interface

Reply:
if we don't pass an assignment logger, validation fails.

That's how I did it initially, but I think we should keep the design agnostic from the provider they're using. WDYT?

Reply:
Oh i see.  yes, you're right.